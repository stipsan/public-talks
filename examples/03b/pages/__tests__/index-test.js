import renderer from 'react-test-renderer'
import App from '../index'

// Mocking the entire component
/*
jest.mock('shared/Footer', () => {
  return {Footer: 'Footer'}
})
//*/

describe('With Snapshot Testing', () => {
  it('App renders correctly', () => {
    const component = renderer.create(<App />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
