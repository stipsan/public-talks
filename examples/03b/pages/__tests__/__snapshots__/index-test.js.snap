// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App renders correctly 1`] = `
<div>
  <p>
    Hello 
    ReactJS Oslo!
    !
  </p>
  <footer>
    Â© 
    2018
     
    ACME Inc.
  </footer>
</div>
`;
